package.json

```diff
@@ -11,7 +11,7 @@
     "build:assets": "yarn --cwd ./projects/assets build",
     "build:cdc": "ng build cdc --prod",
     "build:cds": "ng build cds --prod",
-    "build:libs": "ng build core --prod && ng build storefrontlib --prod && yarn build:schematics && yarn build:organization && yarn build:product && yarn build:product-configurator && yarn build:storefinder && yarn build:qualtrics && yarn build:assets && yarn build:incubator && yarn build:cdc && yarn build:cds && yarn build:setup",
+    "build:libs": "yarn ngcc:lock:remove && ng build core --prod && ng build storefrontlib --prod && yarn build:schematics && yarn build:organization && yarn build:product && yarn build:product-configurator && yarn build:storefinder && yarn build:qualtrics && yarn build:assets && yarn build:incubator && yarn build:cdc && yarn build:cds && yarn build:setup",
```

Comment:
committed on purpose?

Reply:
Done

Reply:
It was discussed with Christoph to remove it as long as it does not work for Windows users. Otherwise, it is not possible to execute **yarn build:libs**